packages/babel-preset/src/assets/normalizeStyleRules.ts

```diff
@@ -22,16 +24,19 @@ export function normalizeStyleRule(
     return ruleValue;
   }
 
-  const result = parseStringWithUrl(ruleValue);
-  // Quotes in URL are optional, so we can also normalize them
-  // https://www.w3.org/TR/CSS2/syndata.html#value-def-uri
-  const url = result.url.replace(/['|"](.+)['|"]/, '$1');
+  return tokenize(ruleValue).reduce((result, token, index, array) => {
```

Comment:
Instead of a regex I am using `tokenize()` from stylis so we can parse all `url()` entries properly.